tests/conftest.py

```
@@ -1129,7 +1131,18 @@ def finalizer():
-                instance.ocp.wait_for_delete(instance.name)
+                log.info(f"Waiting for CephBlockPool {instance.name} to be deleted")
+                instance.ocp.wait_for_delete(
+                    resource_name=instance.name,
+                    timeout=300,
+                )
+                log.info(f"CephBlockPool {instance.name} deleted successfully")
+            except TimeoutError:
+                log.error(f"Timeout waiting for CephBlockPool {instance.name} deletion")
+                log.error(
+                    f"Describe output: {instance.ocp.describe(resource_name=instance.name)}"
+                )
```

Ruff (0.15.17)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)